core/src/test/java/ProcessRunner/ProcessIntegrationTest.java

```diff
@@ -0,0 +1,232 @@
+package ProcessRunner;
+
+
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by Virtuality.
+ */
+class ProcessIntegrationTest {
+
+    @Test
+    void cmdReadTest() {
```

Comment:
Maybe we should skip this before we integrate it into the system? We can't really be platform dependant on windows